docs/dev/table/sql.md

```diff
@@ -2355,6 +2355,41 @@ map ‘[’ key ‘]’
   </tbody>
 </table>
 
+<table class="table table-bordered">
```

Comment:
I would put this under value constructor functions to stay in sync with Calcites documentation (http://calcite.apache.org/docs/reference.html#value-constructors).